### PR TITLE
fix(Error Layout & Footer)

### PR DESCRIPTION
**Pull request by sulha199**
###  Summary

- [ ] Create Generic footer
- [ ] Update User Table's width on screen > tablet
- [ ] Create generic block container
- [ ] Update Error Boundary layout
- [ ] Update Auth UI
- [ ] Show error on pagination

Issue #16

### CODE REVIEW

File: src/images.ts [hunk 1]
```diff
@@ -8,3 +8,10 @@ import "./assets/form-field-error-icon.svg";
 import "./assets/no-users-graphic.svg";
 import "./assets/chevron-left.svg";
 import "./assets/chevron-right.svg";
+import "./assets/right_arrow_icon.svg"
+import "./assets/star_sparkle_stars_sparkles_icon.svg"
+import "./assets/supertokens.svg"
+import "./assets/supertokens-icon.svg"
+import "./assets/supertokens-dark.svg"
```
Comment (nkshah2) [inline src/images.ts:15]: Use images from https://github.com/supertokens/supertokens-logo/tree/master/logo for this.

Dont add the asset files, load the images using the urls

File: src/ui/components/auth/Auth.tsx [hunk 2]
```diff
@@ -16,84 +16,84 @@ import React, { useEffect, useState } from "react";
...
-                        type="text"
-                        error={apiKeyFieldError}
-                        label="API Key"
-                        value={apiKey}
-                        placeholder="Your API Key"
-                    />
+  return (
+    <div className='page-container'>
+      <div className='block-container'>
+        <img className='title-image' src={getImageUrl('star_sparkle_stars_sparkles_icon.svg')} alt='Auth Page' />
```
Comment (nkshah2) [inline src/ui/components/auth/Auth.tsx:77]: Lets rename this image, also when using images from zeplin please rename them

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: src/images.ts (hunk 1 above)
Comment (sulha199) [reply src/images.ts]: its updated

File: src/ui/components/auth/Auth.tsx (hunk 2 above)
Comment (sulha199) [reply src/ui/components/auth/Auth.tsx]: its renamed

**Review by nkshah2 — APPROVED** [review]

**Pull request merged**